python/setup.py

```diff
@@ -298,6 +298,7 @@ def get_outputs(self):
     use_scm_version = {"root": "..", "relative_to": __file__},
     setup_requires=['setuptools_scm'],
     install_requires=['cython >= 0.23', 'numpy >= 1.9', 'six >= 1.0.0'],
+    test_requires=['pytest'],
```

Comment:
No sure what the minimal version is but we left  it out in the `install_requires` as it is an optional dependency. @wesm might know better here.